plugin/sampling/strategystore/adaptive/aggregator_test.go

```diff
@@ -115,3 +115,43 @@ func TestLowerboundThroughput(t *testing.T) {
 	assert.EqualValues(t, 0, a.(*aggregator).currentThroughput["A"]["GET"].Count)
 	assert.Empty(t, a.(*aggregator).currentThroughput["A"]["GET"].Probabilities["0.001000"])
 }
+
+func TestRecordThroughput(t *testing.T) {
+	metricsFactory := metricstest.NewFactory(0)
+	mockStorage := &mocks.Store{}
+	mockEP := &epmocks.ElectionParticipant{}
+	testOpts := Options{
+		CalculationInterval:   1 * time.Second,
+		AggregationBuckets:    1,
+		BucketsForCalculation: 1,
+	}
+	logger := zap.NewNop()
+	a, err := NewAggregator(testOpts, logger, metricsFactory, mockEP, mockStorage)
+	require.NoError(t, err)
+
+	// Testing non-root span
+	span := &model.Span{References: []model.SpanRef{{SpanID: model.NewSpanID(1), RefType: model.ChildOf}}}
+	a.HandleRootSpan(span, logger)
+	require.Empty(t, a.(*aggregator).currentThroughput)
+
+	// Testing span with service name but no operation
+	span.References = []model.SpanRef{}
+	span.Process = &model.Process{
+		ServiceName: "A",
+	}
+	a.HandleRootSpan(span, logger)
+	require.Nil(t, a.(*aggregator).currentThroughput["A"]["GET"])
+
+	// Testing span with service name and operation but no probabilistic sampling tags
+	span.OperationName = "GET"
+	a.HandleRootSpan(span, logger)
+	require.Nil(t, a.(*aggregator).currentThroughput["A"]["GET"])
```

Comment:
```suggestion
	require.Empty(t, a.(*aggregator).currentThroughput)
```
testing for empty is still safer